config/routes.rb

```diff
@@ -39,4 +20,41 @@
   if Rails.env.development?
     mount LetterOpenerWeb::Engine => "letter_opener"
   end
+
+  # Web requests
+  root 'auctions#index'
+  get '/auctions/winners', to: 'auctions#previous_winners'
+
+  get '/auctions/rules/single-bid', to: 'auctions#single_bid_rules'
+  get '/auctions/rules/multi-bid', to: 'auctions#multi_bid_rules'
+
+  # Map current API requests to new controller for now
+  namespace :api, defaults: { format: 'json' } do
+    namespace :v0 do
+      resources :auctions, only: [:index, :show] do
+        resources :bids, only: [:new, :create, :index] do
+          collection do
+            post :confirm
+          end
+        end
+      end
+    end
+  end
+
+  # Temporarily send JSON requests to web to API
```

Comment:
idk maybe when we decide to deprecate "alpha API"? (when versioning is ready / up and running)